package.json

```diff
@@ -182,27 +183,27 @@
     "mui-color-input": "^2.0.1",
     "multicodec": "^3.0.1",
     "multihashes": "^4.0.2",
-    "next": "14.0.4",
+    "next": "15.0.3",
     "next-sitemap": "^4.2.3",
     "node-cache": "^5.1.0",
     "polished": "^4.0.5",
     "polyfill-object.fromentries": "^1.0.1",
     "popper-max-size-modifier": "^0.2.0",
     "qs": "^6.12.1",
-    "react": "18.2.0",
+    "react": "19.0.0-rc-66855b96-20241106",
     "react-appzi": "^1.0.4",
     "react-confetti": "^6.1.0",
     "react-copy-to-clipboard": "^5.0.2",
     "react-cytoscapejs": "^1.2.1",
     "react-device-detect": "^1.6.2",
-    "react-dom": "18.2.0",
+    "react-dom": "19.0.0-rc-66855b96-20241106",
     "react-feather": "^2.0.10",
     "react-ga4": "^1.4.1",
     "react-helmet": "^6.1.0",
     "react-icons": "^5.2.1",
     "react-inlinesvg": "^3.0.1",
     "react-intersection-observer": "^9.10.1",
-    "react-is": "18.2.0",
+    "react-is": "19.0.0-rc-66855b96-20241106",
```

Comment:
This version was set automatically by [codemod](https://react.dev/blog/2024/04/25/react-19-upgrade-guide)